ou_dedetai/config.py

```diff
@@ -59,55 +60,55 @@
     globals()[key] = os.getenv(key, default)
 
 # Set other run-time variables not set in the env.
-ACTION = 'app'
-APPIMAGE_FILE_PATH = None
-authenticated = False
-BADPACKAGES = None
+ACTION: str = 'app'
+APPIMAGE_FILE_PATH: Optional[str] = None
+authenticated = False # FIXME: no references
```

Comment:
This was from a time when I had implemented password requests using the system keyring, authenticated meaning user password authenticated. At the time we chose to drop this functionality for the sake of user trust and we opted for using pkexec in most situations.